drltrace/CMakeLists.txt

```diff
@@ -109,3 +109,12 @@ endif ()
 install(TARGETS drltrace DESTINATION "${INSTALL_BIN}"
   PERMISSIONS ${owner_access} OWNER_EXECUTE GROUP_READ GROUP_EXECUTE
   WORLD_READ WORLD_EXECUTE)
+
+##################################################
+# drltrace test
+
+if (BUILD_TOOL_TESTS)
+  get_target_property(app_path drsyscall_app LOCATION${location_suffix})
+  get_target_property(drltrace_path drltrace LOCATION${location_suffix})
+  add_test(drltrace ${drltrace_path} -- ${app_path})
```

Comment:
Looking.  OK there is only one review comment (thought there were more, prob confusing this w/ the other PR's), but it does request a comment in the cmake for adding more tests (i.e., "XXX i#1960: ....").

nit: also note there is only one colon: https://github.com/DynamoRIO/dynamorio/wiki/Code-Reviews#commit-messages